### PR TITLE
Return vm sizes

### PR DESCRIPTION
**Pull request by acraghav**
Expose an API to return VM sizes from moc-pkg.

### CODE REVIEW

File: services/compute/vmSizes.go [hunk 1]
```diff
@@ -445,3 +453,26 @@ func GetVirtualMachineSizes() (vmsizes *[]VirtualMachineSizeTypes) {
 	vmsizes = &tmp
 	return
 }
+
+func GetVirtualMachineSizesFromMoc() (vMSizes string, err error) {
```
Comment (madhanrm) [inline services/compute/vmSizes.go:457]: Lets not convert this to string here.
Lets return an array of the new type.

Lets rename the method to GetVirtualMachineSizesDetailed

File: go.mod [hunk 2]
```diff
@@ -11,6 +11,7 @@ require (
 	github.com/spf13/viper v1.7.1
 	google.golang.org/genproto v0.0.0-20200526211855-cb27e3aa2013 // indirect
 	google.golang.org/grpc v1.27.1
+	github.com/microsoft/moc-pkg v0.10.13-alpha.1
```
Comment (madhanrm) [inline go.mod:14]: we cannot use moc-pkg. This is a private repo and moc-sdk-for-go is a public repo

File: services/compute/vmSizes.go (hunk 1 above)
Comment (acraghav) [reply services/compute/vmSizes.go]: addressed first 2 comments, 3rd comment I took liberty to name it as GetVirtualMachineSizeValues, let me know if you still don't like, I will rename it back

File: go.mod (hunk 2 above)
Comment (acraghav) [reply go.mod]: done. moved the constants to moc-common

**Review by madhanrm — APPROVED** [review]

**Comment by madhanrm** [comment]
can you revendor this in moccli and update the cmdlet

https://github.com/microsoft/moccli/blob/master/cmd/mocctl/compute/virtualmachine/list_sizes/virtualmachine_list_sizes.go

**Pull request merged**